src/components/DataBrowser.vue

```diff
@@ -324,21 +324,21 @@ girder-data-table.girder-file-browser(
     @dragstart="$emit('dragstart', $event)",
     @dragend="$emit('dragend', $event)",
     @drop="$emit('drop', $event)")
-
   template(#header="{ props, on }")
-    tr.secondary(:class="$vuetify.theme.dark?'darken-2':'lighten-5'")
```

Comment:
This was just a collateral fix.  At some point in the past, this slot became responsible for adding the thead tag.  I don't think it makes a difference.